blocks/table-of-contents/src/script.js

```diff
@@ -50,14 +51,17 @@ function handleBlock( block ) {
 		}
 	});
 
+
 	/**
 	 * Render
 	 * data -> links -> listItems -> list -> nav
 	 */
+	renderData = data
```

Comment:
Can this either be assigned using `const` or put back inside `renderList` so it doesn't need to be assigned to a variable?